DevelopGitHub/GitHubExtension/GitHubExtension.Activity.WebApi/Services/Interfaces/IActivityWriterService.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using GitHubExtension.Activity.Internal.DAL;
+
+namespace GitHubExtension.Activity.Internal.WebApi.Services.Interfaces
+{
+    public interface IActivityWriterService
```

Comment:
Writer and reader do a mix things....May be it will be better use commands and queries? Remember CQRS ?